format/applebookmark/apple_bookmark.go

```diff
@@ -0,0 +1,311 @@
+package applebookmark
+
+import (
+	"embed"
+	"time"
+
+	"github.com/wader/fq/format"
+	"github.com/wader/fq/pkg/decode"
+	"github.com/wader/fq/pkg/interp"
+	"github.com/wader/fq/pkg/scalar"
+)
+
+//go:embed apple_bookmark.jq apple_bookmark.md
+var bookmarkFS embed.FS
+
+func init() {
+	interp.RegisterFormat(decode.Format{
+		Name:        format.APPLE_BOOKMARK,
+		ProbeOrder:  format.ProbeOrderBinUnique,
+		Description: "Apple BookmarkData",
+		Groups:      []string{format.PROBE},
```

Comment:
Does it make sense to make the format probable? are the places were it's used in separate files? thinking if we will have some jq code that explicitly decodes it's not needed.

Reply:
I haven't seen them outside of the `bplist` test files I've been working with, but they can be created as part of Apple's public APIs so it's possible that they might be? Some quick googling didn't tell me much about how other people are storing these.

Reply:
Okok, i'm mostly concerned about have decoders in the probe group if they can mis-probe or for some reason will take long time to fail slowing down probing a lot

Reply:
Let's keep in probe for now, as it assert early on book/alia it feels kind of safe